preview-src/pullRequestOverviewRenderer.ts

```diff
@@ -227,57 +227,104 @@ function groupBy<T>(arr: T[], fn: (el: T) => string): { [key: string]: T[] } {
+	const reviewCommentContainer = document.createElement('div');
+	reviewCommentContainer.className = 'review-comment-container';
+	commentContainer.appendChild(userIcon);
+	commentContainer.appendChild(reviewCommentContainer);
+
+	const commentHeader = document.createElement('div');
+	commentHeader.className = 'review-comment-header';
+	const authorLink = document.createElement('a');
+	authorLink.className = 'author';
+	(<HTMLAnchorElement>authorLink).href = comment.user.html_url;
+	authorLink.textContent = comment.user.login;
+
+	const timestamp = document.createElement('div');
+	timestamp.className = 'timestamp';
+	timestamp.textContent = moment(comment.created_at).fromNow();
 
+	const commentBody = document.createElement('div');
+	commentBody.className = 'comment-body';
+	commentBody.innerHTML  = md.render(emoji.emojify(comment.body));
+
+	commentHeader.appendChild(authorLink);
+	commentHeader.appendChild(timestamp);
+
+	reviewCommentContainer.appendChild(commentHeader);
+	reviewCommentContainer.appendChild(commentBody);
+
+	return commentContainer;
+}
+
+export function renderCommit(timelineEvent: CommitEvent): HTMLElement {
 	const shaShort = timelineEvent.sha.substring(0, 7);
-	const avatar = timelineEvent.author.avatar_url
-		? `<div class="avatar-container"><a class="avatar-link" href="${timelineEvent.author.html_url}"><img class="avatar" src="${timelineEvent.author.avatar_url}"></a></div>`
-		: '';
-	const login = timelineEvent.author.login
-		? `<a class="author" href="${timelineEvent.author.html_url}">${timelineEvent.author.login}</a>`
-		: timelineEvent.author.name;
-
-	return `<div class="comment-container"  data-type="commit">
-
-	<div class="review-comment" role="treeitem">
-		<div class="review-comment-contents commit">
-			<div class="commit">
-				<div class="commit-message">
-					<svg class="octicon octicon-git-commit" width="14" height="16" viewBox="0 0 14 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-						<path fill-rule="evenodd" clip-rule="evenodd" d="M10.86 3C10.41 1.28 8.86 0 7 0C5.14 0 3.59 1.28 3.14 3H0V5H3.14C3.59 6.72 5.14 8 7 8C8.86 8 10.41 6.72 10.86 5H14V3H10.86V3ZM7 6.2C5.78 6.2 4.8 5.22 4.8 4C4.8 2.78 5.78 1.8 7 1.8C8.22 1.8 9.2 2.78 9.2 4C9.2 5.22 8.22 6.2 7 6.2V6.2Z" transform="translate(0 4)"/>
-					</svg>
-					${avatar}
-					<div class="message">
-						${login} ${timelineEvent.message}
-					</div>
-				</div>
-				<a class="sha" href="${timelineEvent.html_url}">${shaShort}</a>
-			</div>
-		</div>
-	</div>
-</div>`;
+
+	const commentContainer = document.createElement('div');
+	commentContainer.classList.add('comment-container', 'commit');
+	const commitMessage = document.createElement('div');
+	commitMessage.className = 'commit-message';
+
+	const commitIcon = document.createElement('span');
+	commitIcon.innerHTML = `<svg class="octicon octicon-git-commit" width="14" height="16" viewBox="0 0 14 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe this could go in `media`?

Reply:
I moved it to `resources` instead, since it looks like `media` is the output directory for webpack